python/bridgestan/model.py

```diff
@@ -36,9 +37,10 @@ def from_param(cls, obj):
 
 FloatArray = Union[
     npt.NDArray[np.float64],
-    ctypes.POINTER(ctypes.c_double),
+    "ctypes._Pointer[ctypes.c_double]",
```

Comment:
Why the change here?

Reply:
Two parter:

ctypes.POINTER is a value, not a type (like saying we had a Union of 1, vs int). My bad

ctypes._Pointer is the corresponding type, but it is not a real generic at runtime, hence why it is quotes